packages/eos-components/src/components/Card.vue

```diff
@@ -33,6 +32,19 @@ export default {
           return 'RegularCard';
       }
     },
+    subtitle() {
+      let fallback = '';
+      if (this.$store) {
+        const { state } = this.$store;
```

Comment:
Oh, this is a great way to use the vuex store from the reusable components. I've been investigating another approach: https://dev.to/blankazucenalg/como-utilizar-un-proyecto-de-vue-como-biblioteca-de-componentes-4ko